apps/mobile-wallet/src/components/headers/ProgressHeader.tsx

```diff
@@ -19,7 +19,7 @@ const workflowSteps: Record<
   (keyof ReceiveNavigationParamList)[] | (keyof SendNavigationParamList)[] | (keyof BackupMnemonicNavigationParamList)[]
 > = {
   receive: ['AddressScreen', 'QRCodeScreen'],
-  send: ['DestinationScreen', 'OriginScreen', 'AssetsScreen', 'VerifyScreen'],
+  send: ['DestinationScreen', 'OriginScreen', 'AddressTokensScreen', 'VerifyScreen'],
```

Comment:
We agreed in the past to get rid of the `Assets` type and naming in our code since everything is a "token", but maintain the word "assets" when it comes to translatable texts.